src/pages/home/report/MarkerBadge/MarkerBadgeContainer/MarkerBadgeContainerPropTypes.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Container Styles */
+    containerStyles: PropTypes.arrayOf(PropTypes.object).isRequired,
+
+    /** Children of the MakerBadgeContainer */
```

Comment:
NAB; What do you think about putting "Rendered child component" here? I think this is slightly more useful

Reply:
yeah Better